pkg/ceph/mon/config.go

```diff
@@ -58,7 +58,7 @@ type cephConfig struct {
 }
 
 type GlobalConfig struct {
-	EnableExperimental       string `ini:"enable experimental unrecoverable data corrupting features"`
+	EnableExperimental       string `ini:"enable experimental unrecoverable data corrupting features,omitempty"`
```

Comment:
Doesn't this "remove" the `EnableExperimental` option in the config when empty?

Reply:
that's correct, we shouldn't need the property in the config since bluestore isn't experimental anymore